src/client/client.cpp

```diff
@@ -1195,7 +1188,7 @@ void Client::clearOutChatQueue()
 }
 
 void Client::sendChangePassword(const std::string &oldpassword,
-        const std::string &newpassword)
+		const std::string &newpassword)
```

Comment:
https://dev.minetest.net/Code_style_guidelines#Function_declarations
>In case your function parameters don't fit within the defined line length use following style. Indention for follow up lines is **exactly** one tab.

Reply:
Sorry, cap'n. Code-style's now fixed. O7 :medal_military: